packages/checkout/widgets-lib/src/widgets/sale/SaleWidgetRoot.tsx

```diff
@@ -132,6 +138,9 @@ export class Sale extends Base<WidgetType.SALE> {
                   excludePaymentTypes={this.parameters.excludePaymentTypes!}
                   language="en"
                   multicurrency={!!this.properties?.config?.multicurrency}
+                  waitFulfillmentSettlements={
```

Comment:
Question and suggestion?
I think we may want the wait to be enabled by default, right? If so then I suggest you use
```
waitFulfillmentSettlements={this.properties?.config?.waitFulfillmentSettlements ?? true}
```
this expresion will allow`false`, but if null or undefined will default to `true`